tests/test_station_data_interface.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+import pytest
+
+from krakus.utils.wios import StationDataInterface
+
+
+def test_title(wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    assert interface.title == wios_station_data['data']['title']
+    assert isinstance(interface.title, str)
+
+
+def test_data(wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    assert interface.data == wios_station_data['data']
+    assert isinstance(interface.data, dict)
+
+
+def test_series(wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    assert interface.series == wios_station_data['data']['series']
+    assert isinstance(interface.series, list)
+
+
+def test_filter_with_empty_series(wios_station_data):
+    wios_station_data['data']['series'] = []
+    interface = StationDataInterface(wios_station_data)
+    assert interface._filter_series('pm10') == []
+
+
+def test_filter_bad_key(wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    assert interface._filter_series('foo') == []
+
+
+@pytest.mark.parametrize('attr', ['pm10_data', 'pm25_data'])
+def test_pm_data(attr, wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    # for some reasons there are 26 measurements for one single day
```

Comment:
Maybe this is due to fact that we're using CET (UTC+1h) and CEST (UTC+2h) in Poland?

Reply:
probably